### PR TITLE
add lava to keplr

### PR DESCRIPTION
**Pull request by omerlavanet**

### CODE REVIEW

**Comment by Thunnini** [comment]
I will merge when lava mainnet starts.

**Comment by JohnnWi** [comment]
> I will merge when lava mainnet starts.

and when will start? thanks

File: cosmos/lava.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    "bech32PrefixValPub": "lava@valoperpub",
+    "bech32PrefixConsAddr": "lava@valcons",
+    "bech32PrefixConsPub": "lava@valconspub"
+  },
+  "currencies": [
+    {
+      "coinDenom": "LAVA",
+      "coinMinimalDenom": "ulava",
+      "coinDecimals": 6,
+      "coinGeckoId": "lava-network",
```
Comment (editaahn) [inline cosmos/lava.json:32]: Is it going to be available at the moment when it's launched? If not, please remove the coinGeckoId lines in `currencies`, `feeCurrencies` and `stakeCurrency` for this PR and add it afterwards.

File: cosmos/lava.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  "chainName": "Lava",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/lava/chain.png",
+  "stakeCurrency": {
+    "coinDenom": "LAVA",
+    "coinMinimalDenom": "ulava",
+    "coinDecimals": 6,
+    "coinGeckoId": "lava-network",
+    "coinImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/lava/ulava.png"
+  },
+  "walletUrl": "https://lavanet.xyz",
```
Comment (editaahn) [inline cosmos/lava.json:14]: Could you remove this line as well? walletUrl is a deprecated property.

**Review by editaahn — COMMENTED** [review]
Hi! Please pull the current changes from the `main` branch. There were changes on the validations.
Also, make sure the image sizes are 256*256px.
Thank you.

File: cosmos/lava.json (hunk 2 above)
Comment (omerlavanet) [reply cosmos/lava.json:14]: done

File: cosmos/lava.json (hunk 1 above)
Comment (omerlavanet) [reply cosmos/lava.json:32]: yes, there's already an entry without a price, and it will be reflected on mainnet, we work with coingecko

**Pull request merged**